engines/bastion_katello/app/assets/javascripts/bastion_katello/products/discovery/discovery.controller.js

```diff
@@ -114,10 +114,11 @@ angular.module('Bastion.products').controller('DiscoveryController',
                 params = {
                     url: $scope.discovery.url,
                     label: '',
-                    contentType: $scope.discovery.contentType
+                    contentType: $scope.discovery.contentType,
+                    repositoryUrl: url
                 };
                 if ($scope.discovery.contentType === 'yum') {
-                    params.path = url.replace(baseUrl, "");
+                    params.path = new URL(url).pathname.replace(new URL(baseUrl).pathname, "/");
```

Comment:
This is a little hard to read, it could help to break it up to:
```javascript
var urlPath = new URL(url).pathname
var baseUrlPath = new URL(baseUrl).pathname
params.path = urlPath.replace(baseUrlPath, "/");
```

but just a suggestion, not a blocker or anything :)